frontend/src/core/comments/components/TeamComment.css

```diff
@@ -0,0 +1,6 @@
+.team-comment > p {
+    border-bottom: 1px solid #5E6475;
+    color: #aaa;
+    font-style: italic;
+    padding: 15px 10px;
+}
```

Comment:
Please make sure this is styled exactly the same as in other two tabs. Right now it is not - I see that the `line-height` is not the same as in other-locales.